docs/connect.rst

```diff
@@ -0,0 +1,245 @@
+.. _connect:
+
+==================
+Connect to CrateDB
+==================
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+.. _data-source-name:
+
+Data Source Names
+=================
+
+PDO makes use of `Data Source Name`_ (DSN) strings.
+
+A basic version of the DSN string for CrateDB might be::
+
+    crate:<HOST_ADDR>:<PORT>
+
+Here, replace ``<HOST_ADDR>`` with the host address of a CrateDB node, and
+replace ``<PORT>`` with the port number of the  `HTTP endpoint`_.
+
+Example DSN strings:
+
+- ``crate:localhost:4200``
+- ``crate:crate-1.vm.example.com:4200``
+- ``crate:198.51.100.1:4200``
+
+You can specify a second CrateDB node, like so::
+
+    crate:<HOST_ADDR_1>:<PORT>,<HOST_ADDR_2>:<PORT>
+
+Here, ``<HOST_ADDR_1>`` and ``<HOST_ADDR_2>`` are the host strings for the
+first and second CrateDB nodes, respectively.
+
+In fact, you can specify as many nodes as you like. Each corresponding host
+string must be separated from the previous one using a ``,`` character.
+
+The driver will attempt to connect to each node in the order they appear. The
```

Comment:
that's not correct, one hosts is chosen by random (https://github.com/crate/crate-pdo/blob/master/src/Crate/PDO/Http/ServerPool.php#L79) to not always hit the same node.